src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Authentication.cs

```diff
@@ -188,7 +188,9 @@ public async Task HttpClientHandler_IncorrectCredentials_Fails(string authentica
             {
                 HttpClientHandler handler = CreateHttpClientHandler();
                 Task serverTask = server.AcceptConnectionPerformAuthenticationAndCloseAsync(authenticateHeader);
-                await TestHelper.WhenAllCompletedOrAnyFailed(CreateAndValidateRequest(handler, url, HttpStatusCode.Unauthorized, new NetworkCredential("wronguser", "wrongpassword")), serverTask);
+
+                NetworkCredential credentials = TestSecretHelper.CreateTestCredential("wronguser", "wrongpassword");
```

Comment:
The tooling doesn't flag this?

Reply:
No, it doesn't. The rule is searching for `new NetworkCredential` +  password in string literal form